src/main/backend/process.ts

```diff
@@ -28,6 +28,7 @@ const getBackendPath = lazy((): string => {
 interface BaseBackendProcess {
     readonly owned: boolean;
     readonly port: number;
+    readonly url: string;
```

Comment:
I think we should remove `port`. `port` assumed localhost, so what's the point? Anything that uses `port` is incorrect anyway.